### ISSUE TITLE
[ux fix] put the type prtification after the default type suggestion

### ISSUE THREAD

**Issue by wael-boss** [mylesmmurphy/prettify-ts#34]
sometimes the default abriviated straight to the point type suggestion is what you need so put it first or at least a setting to put it first then for a more detailed type prview the user can just scroll for your pritification

**Comment by mylesmmurphy** [comment]
Hey @wael-boss!
Unfortunately VSCode does not expose a method to determine the order of hover providers, however, in the future this extension might integrate directly with the default type preview, and in that case I can add code to add this. I will add this to the future roadmap

**Comment by vladyslav-n** [comment]
Yep, I'd say that putting the default type synopsis in the beginning would have been an amazing improvement!

**Comment by qaws5503** [comment]
If this feature can be published, this extension will be a game killer🔪🔪

**Comment by denis-migdal** [comment]
TS now has expandable hover.

If we can't change the hover order, could it be possible to somehow make the prettified type hover collapsable ?
Maybe it could be possible to increase and reduce the level of details of the prettified type with 2 buttons (+/-) ?
